sunspot/spec/mocks/connection.rb

```diff
@@ -124,5 +128,9 @@ def with?(request, params)
         request.has_key?(params)
       end
     end
+
+    def commitWithinValue(opts = {})
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use snake_case for method names.](https://www.codacy.com/app/joni-amir/sunspot/file/2482066397/issues/source?bid=3265104&fileBranchId=3265120#l132)